main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"github.com/luciodesimone/golang-bootcamp/db"
+	"log"
+)
+
+func main() {
+	stout := db.Beer{
+		Desc:        "Creamy black dense beer with coffee notes, ideal before dinner",
+		IBU:         42,
+		AlcoholCont: 8,
+		SRM:         40,
+		AvgScore:    92,
+	}
+
+	bitter := db.Beer{
+		Desc:        "dark amber light beer, ideal for summer",
+		IBU:         57,
+		AlcoholCont: 5.5,
+		SRM:         10,
+		AvgScore:    63,
+	}
+
+	barley := db.Beer{
+		Desc:        "Strong, dense red/dark beer, ideal for late night with friends",
+		IBU:         70,
+		AlcoholCont: 12,
+		SRM:         22,
+		AvgScore:    86,
+	}
+
+	c := db.Connection()
+
+	s := c.Create(stout)
+	bi := c.Create(bitter)
+	ba := c.Create(barley)
+
+	log.Printf("Created: %v", s)
+	log.Printf("Created: %v", bi)
+	log.Printf("Created: %v", ba)
+
+	s = c.Get(1)
```

Comment:
`s` is of type interface{}. I saw that on the Description of the PR you correctly noted that there were no type assertions done. 

But from a concept point of view, the DB is returning "anything" and the user of that DB cannot know before hand what that thing is.

For the sake of simplicity, we could make a DB that just stores Beer. We can of course take a look at this together.